android/anc/src/main/java/org/smartregister/fhircore/anc/ui/details/PatientDetailsActivity.kt

```diff
@@ -140,8 +164,42 @@ class PatientDetailsActivity : BaseMultiLanguageActivity() {
           .joinToString(separator = " ID: ")
       activityAncDetailsBinding.txtViewPatientDetails.text = patientDetails
       activityAncDetailsBinding.txtViewPatientId.text = patientId
-      isPregnant =
-        if (this.patientDetails.gender == "Male") true else this.patientDetails.isPregnant
+      isMale = this.patientDetails.gender == "Male"
```

Comment:
Is `gender` always uppercase? I think we should create an `isMale(patientDetails)` helper function so we can consolidate any logic related to male determination in a single location

Reply:
@pld in this case we extracting gender from Patient resource and saving as "Male", "Female", "Unknown"

Reply:
Ah yes that's exactly the challenge with this, the change you made looks great, that's a sufficient level of abstraction for the time being, if in the feature we are checking this in multiple places we can add an `isPatientMale` helper function or something like that but we don't need it now